plugins/obs-filters/mask-filter.c

```diff
@@ -220,6 +225,9 @@ static void mask_filter_render(void *data, gs_effect_t *effect)
 	param = gs_effect_get_param_by_name(filter->effect, "color");
 	gs_effect_set_vec4(param, &filter->color);
 
+	param = gs_effect_get_param_by_name(filter->effect, "opacity");
+	gs_effect_set_int(param, filter->opacity);
```

Comment:
opacity should be a float and scaled on the CPU (divide by 100.0f).

Reply:
I favor multiplication over division. However, this does make me believe that "Scaling of opacity should be done on the CPU, not GPU." comments are regarding this.

Making changes.